File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/EURBuyComparator.java
package fobo66.exchangecourcesbelarus.util.comparators;

import fobo66.exchangecourcesbelarus.models.Currency;

/**
 * Created by fobo66 on 31.03.2016.
 */
public class EURBuyComparator implements CurrencyComparator {
    @Override
    public int compare(Currency lhs, Currency rhs) {
        return rhs.eurBuy.compareToIgnoreCase(lhs.eurBuy);
    }
}

File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/EurBuyComparator.kt
package fobo66.exchangecourcesbelarus.util.comparators

import fobo66.exchangecourcesbelarus.entities.Currency

/**
 * Created by fobo66 on 31.03.2016.
 */
class EurBuyComparator : CurrencyComparator {
  override fun compare(lhs: Currency, rhs: Currency): Int {
    return rhs.eurBuy.compareTo(lhs.eurBuy, ignoreCase = true)
  }
}
